src/tracks/playabletrack/wavetrack/ui/PitchAndSpeedDialog.cpp

```diff
@@ -82,6 +82,18 @@ class ScopedVerticalLay : public ScopedSizer<void, int>
    }
 };
 
+class ScopedStatic :
+    public ScopedSizer<wxStaticBox*, const TranslatableString&, int>
+{
+public:
+   ScopedStatic(ShuttleGui& s, const TranslatableString& label, int iProp = 0)
+       : ScopedSizer<wxStaticBox*, const TranslatableString&, int>(
+            s, &ShuttleGui::StartStatic, &ShuttleGui::EndStatic,
+            std::move(label), std::move(iProp))
```

Comment:
`label` should be passed by value then, not reference.
`move` on `iProp` is redundant

Reply:
It wasn't when I wrote that line, but that was because of the `Args&&...` in the class template. I removed `&&` and now I don't have to move the ints anymore. I think it's because before it expected right-reference types.